scripts.js

```diff
@@ -0,0 +1,12 @@
+var a = 2,
+	b = 4,
+	value = (a * a) - (2 * a * b) + (b * b);
+console.log(value);
+if (value > 0) {
+	console.log('wynik dodatni');
+} else if (value < 0) {
+	console.log('wynik ujemny');
+} else {
```

Comment:
are you sure that value in this case always will be equal to 0? what will happen if variable a will be equal to letter?